src/spartan/spec.clj

```diff
@@ -738,12 +771,15 @@
   (let [id (gensym)
         predx #(let [mm mm]
                  (mm %))]
-    {:type ::spec
+    {:type   ::spec
      :cform  (fn [_ x]
                (if-let [pred (predx x)]
                  (dt pred x nil #_form)
-                 ::invalid))}))
-
+                 ::invalid))
+     :unform (fn [_ x]
+               (if-let [pred (predx x)]
+                         (unform pred x)
+                         (throw (Exception. (str "No method of: " form " for dispatch value: " #_ (dval x))))))}))
```

Comment:
I didn't figure how to make `dval` implementation on Babashka, original code:

```clojure
dval #((.dispatchFn ^clojure.lang.MultiFn @mm) %)]
```

Reply:
Also, the original exception here was a `IllegalStateException`, Babashka tells its not found, is it fine to use the `Exception` here instead?